docs/development/getting_started.rst

```diff
@@ -1,163 +1,190 @@
-to a virtual environment using ``virtualenvwrapper`` and ``pip``, so as not to
-install the older version we use system-wide. The following commands assume your
-default Python is the Python2 that ships with macOS. If you are using a
-different version, the path to ``virtualenvwrapper.sh`` will differ. Running
-``pip show virtualenvwrapper`` should help you find it.
-
-.. code:: sh
-
-    sudo easy_install pip # if you don't already have pip
-    pip install -U virtualenvwrapper
-    source /usr/local/bin/virtualenvwrapper.sh
-    mkvirtualenv -p python2 securedrop
-
-.. note:: You'll want to add the command to source ``virtualenvwrapper.sh``
-          to your ``~/.bashrc`` (or whatever your default shell configuration
-          file is) so that the command-line utilities ``virtualenvwrapper``
-          provides are automatically available in the future.
-
-.. _Vagrant: http://www.vagrantup.com/downloads.html
-.. _VirtualBox: https://www.virtualbox.org/wiki/Downloads
-.. _Ansible: http://docs.ansible.com/intro_installation.html
-.. _Homebrew: https://brew.sh/
-.. _Docker: https://store.docker.com/editions/community/docker-ce-desktop-mac
-
-Fork & Clone the repository
----------------------------
-
-Now you are ready to get your own copy of the source code.
-Visit our repository_ fork it and clone it on you local machine:
-
-
-.. code:: sh
-
-   git clone git@github.com:<your_github_username>/securedrop.git
-
-.. _repository: https://github.com/freedomofpress/securedrop
-
-Install python requirments
---------------------------
-
-SecureDrop uses many 3rd party open source packages from the python community.
-Ensure your virtualenv is activated and install the packages.
-
-.. code:: sh
-
-    pip install -r securedrop/requirements/develop-requirements.txt
-
-.. note:: You will need to run this everytime new packages are added.
+Contributing to SecureDrop
+==========================
+
+If you have time, one hour or a year, we welcome your help in various
+areas. We are commited to make it as easy as possible for you to
+contribute.
+
+* `Programmers`_, to help us developing SecureDrop.
+* `DevOps`_, to work on the community maintained SecureDrop infrastructure.
+* `Release managers`_, to create and maintain Debian GNU/Linux packages and repositories.
+* `Writers`_, to help improve the documentation.
+* `Translators`_, to translate SecureDrop
+* `Moderators & Support`_, to answer on `the forums <https://forum.securedrop.club/c/support>`__.
+* `Designers`_, for SecureDrop source and journalist web interfaces and Tails customization.
+
+You will find a a developer to answer any questions you may have at
+the `SecureDrop instant messenging channel
+<https://gitter.im/freedomofpress/securedrop>`__, whatever your
+timezone is. You should also register to `the forum
+<https://forum.securedrop.club/>`__ for longer discussions and to look
+for answers.
+
+Programmers
+~~~~~~~~~~~
+
+SecureDrop is a framework including Flask based web applications for
```

Comment:
The phrases `SecureDrop is a free software product`, `SecureDrop is a software solution` or simply `the SecureDrop system` are more accurate than `SecureDrop is a framework` here (since framework usually refers to a software product intended for developers as the consumer).